tests/unit/readers/test_skoscodelistreader.py

```diff
@@ -13,70 +13,67 @@
-     codelist CSV-W.
+    Ensure that we can successfully extract the concept scheme URI and concept URI template from a roughly standard codelist CSV-W with multiple tables.
     """
     code_list_csvw = (
-        _test_case_base_dir / "utils" / "csvw" / "code-list.csv-metadata.json"
+        _test_case_base_dir / "utils" / "csvw" / "industry-grouping.csv-metadata.json"
     )
     csv_path, cs_uri, concept_uri_template = extract_code_list_concept_scheme_info(
         code_list_csvw
     )
 
-    assert csv_path == "code-list.csv"
-    assert cs_uri == "http://gss-data.org.uk/def/trade/concept-scheme/age-of-business"
+    assert csv_path == "industry-grouping.csv"
+    assert (
+        cs_uri
+        == "http://gss-data.org.uk/data/gss_data/trade/ons-international-trade-in-services-by-subnational-areas-of-the-uk#scheme/industry-grouping"
+    )
     assert (
         concept_uri_template
-        == "http://gss-data.org.uk/def/trade/concept/age-of-business/{+notation}"
+        == "http://gss-data.org.uk/data/gss_data/trade/ons-international-trade-in-services-by-subnational-areas-of-the-uk#concept/industry-grouping/{+notation}"
     )
 
 
-def test_skos_in_scheme_missing():
+def test_skos_concept_scheme_csvw():
     """
-    Ensure that we get an appropriate exception when the `skos:inScheme` triple column is missing.
+    Ensure that we can successfully extract the concept scheme URI and concept URI template from a roughly standard codelist CSV-W.
     """
     code_list_csvw = (
-        _skos_codelist_reader_test_cases / "skos-in-scheme-missing.csv-metadata.json"
+        _test_case_base_dir / "utils" / "csvw" / "code-list.csv-metadata.json"
     )
-
-    with pytest.raises(ValueError) as ex:
-        extract_code_list_concept_scheme_info(code_list_csvw)
-
-    assert "is missing `skos:inScheme` column" in str(ex)
-
-
-def test_about_url_missing():
-    """
-    Ensure that we get an appropriate exception when the concept aboutUrl is missing.
-    """
-    code_list_csvw = (
-        _skos_codelist_reader_test_cases / "about-url-missing.csv-metadata.json"
+    csv_path, cs_uri, concept_uri_template = extract_code_list_concept_scheme_info(
+        code_list_csvw
     )
 
-    with pytest.raises(ValueError) as ex:
-        extract_code_list_concept_scheme_info(code_list_csvw)
-
-    assert "is missing `aboutUrl` property" in str(ex)
+    assert csv_path == "code-list.csv"
+    assert cs_uri == "http://gss-data.org.uk/def/trade/concept-scheme/age-of-business"
+    assert (
+        concept_uri_template
+        == "http://gss-data.org.uk/def/trade/concept/age-of-business/{+notation}"
+    )
 
 
-def test_csv_url_missing():
+def test_num_of_variables_in_about_url_invalid_exception():
     """
-    Ensure that we get an appropriate exception when the table's URL is missing.
+    Ensures that an exception is thrown when the number of variables in about url are invalid.
```

Comment:
I have a vague memory of us wanting to support multiple variable names, was the code set up to allow this before?

Reply:
@robons I know that the `qbwriter` has some code for supporting multiple variables in the same url (see `_get_default_value_uri_for_code_list_concepts` in `qbwriter.py`). However, the existing code around `skoscodelistreader` or any code I have refactored was not supporting this as far as I can see.

Reply:
Yeah, you're right, I may just have been thinking about the `about_url` defined somewhere else.